src/LaboratoryResults-3-0/Test/XIS-Server/TEST-laboratoryresults-LAB-LRO-2-1-receive-panelLOINC.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="server">
+  
+    <id value="TEST-xis-2-1-Receive-panel"/>
+    <version value="R4-3.0"/>
+    <name value="LaboratoryResults - LAB-LRO - Scenario 2.1 - Receive a panel of laboratory results with a LOINC code."/>
+    <description value="Scenario 2.1 - Receive a panel of laboratory results with a LOINC code."/>
+ 
+    <nts:fixture id="TEST-scenario-2-1-panel-fixture"
+        href="resources/lr-slr-TEST-Scenario1b-1b.xml"/>
```

Comment:
Van dit XIS script worden automatisch een XML en een JSON versie gemaakt. Doordat hier `.xml` staat, wordt door Touchstone ook in de JSON-test een XML-fixture gemaakt.

Om dit op te lossen moeten 2 dingen gebeuren:
1. Hier moet komen te staan: `<nts:fixture id="TEST-scenario-2-1-panel-fixture" href="resources/lr-slr-TEST-Scenario1b-1b.{$_FORMAT}"/>`. Dan wordt in de output automatisch `.xml` en `.json` neergezet.
2. De betreffende Bundles moeten omgezet worden naar JSON. Dit kan met de HL7 validator (https://confluence.hl7.org/display/FHIR/Using+the+FHIR+Validator). `java -jar C:\util\validator_cli.jar lr-slr-TEST-Scenario1b-1b.xml -version 4.0.1 -convert -output lr-slr-TEST-Scenario1b-1b.json`

Reply:
En dit voor iedere Receive test